tools/check/check-gogenerate.sh

```diff
@@ -2,7 +2,9 @@
 set -euo pipefail
 
 go generate ./...
+set +e
 diffline=$(git status -s | awk '{print $2}' | xargs grep '^// Code generated .* DO NOT EDIT\.$' 2>/dev/null | wc -l)
```

Comment:
How about `diffline=$(git status -s | awk '{print $2}' | xargs grep '^// Code generated .* DO NOT EDIT\.$' 2>/dev/null || true | wc -l)`?